hatch.toml

```diff
@@ -19,7 +19,11 @@ features = ["test", "dask-ml"]
 extra-dependencies = ["ipykernel"]
 overrides.matrix.deps.env-vars = [
     { if = ["pre"], key = "UV_PRERELEASE", value = "allow" },
-    { if = ["min"], key = "UV_RESOLUTION", value = "lowest-direct" },
+    #{ if = ["min"], key = "UV_RESOLUTION", value = "lowest-direct" },
+    { if = ["min"], key = "UV_CONSTRAINT", value = "/tmp/scanpy-min-deps.txt" },
```

Comment:
1. Delete commented out?
2. Why not put this in the `ci` folder or something and then `.gitignore` it?  I don't know enough about filesystems, but I seem to remember Helmholtz's having no accessible `tmp`

Reply:
can do!